<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Xiugaimima</class>
 <widget class="QWidget" name="Xiugaimima">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校园寻路系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="School.qrc">
    <normaloff>:/School/Picture/tb.png</normaloff>:/School/Picture/tb.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/School/Picture/xgmm.png);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="Wangji_Button">
     <property name="font">
      <font>
       <family>方正粗黑宋简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>忘记密码</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLineEdit" name="Yhm_Line">
     <property name="font">
      <font>
       <family>方正粗黑宋简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLineEdit" name="Mm_line">
     <property name="font">
      <font>
       <family>方正粗黑宋简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>方正粗黑宋简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入修改的密码：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>方正粗黑宋简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>忘记密码？</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="Fanhui_Button">
     <property name="font">
      <font>
       <family>方正粗黑宋简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="QueRen_Button">
     <property name="font">
      <font>
       <family>方正粗黑宋简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>方正粗黑宋简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入用户名：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLineEdit" name="Wjmm_line">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>方正粗黑宋简体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="School.qrc"/>
 </resources>
 <connections/>
</ui>
